src/Microsoft.Diagnostics.Monitoring.Options/MonitorCapability.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel.DataAnnotations;
+using System.Text.Json.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.Options
+{
+    public class MonitorCapability
+    {
+        [Required]
+        [JsonPropertyName("name")]
+        public string Name { get; set; } = string.Empty;
```

Comment:
If this is required, then let's remove the default value and put a `required` keyword on it.